tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
```
@Foo => { 'A' => ['foo', 'bar']}
@Bar => { 'A' => ['bar']}
@Foo - @Bar => { 'A' => ['foo']}
```
the issue with it is, that for `A` both `[foo, bar]` and `[bar]` might be valid config, however this doesn't guarantee `[foo]` is a valid option. This adds another layer of complexity of which I wonder is worth to resolve.

If we would only remove rules if the config match exactly, than we are stuck with some odd cases.
Like;
```
@SF{'header' => 'some SF copyright statment'}
-
@CommentFixers{'header' => false}
```

would still result in adding the header, which I think is odd.